ddtrace/internal/datadog/profiling/build_standalone.sh

```diff
@@ -103,6 +103,7 @@ cmake_args=(
   -DCMAKE_EXPORT_COMPILE_COMMANDS=ON
   -DCMAKE_VERBOSE_MAKEFILE=ON
   -DLIB_INSTALL_DIR=$(realpath $MY_DIR)/lib
+  -DPython3_EXECUTABLE=$(which python3)
```

Comment:
Out of curiosity, what's the purpose of this change? The PR description doesn't mention it. Was CMake choosing something other than the `python3` in your PATH? The docs are a little vague (to me) on how the executable is chosen: https://cmake.org/cmake/help/latest/module/FindPython3.html#hints